ext/standard/basic_functions_arginfo.h

```diff
@@ -875,6 +875,11 @@ ZEND_BEGIN_ARG_WITH_RETURN_TYPE_MASK_EX(arginfo_stristr, 0, 2, MAY_BE_STRING|MAY
 	ZEND_ARG_TYPE_INFO(0, before_needle, _IS_BOOL, 0)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_MASK_EX(arginfo_str_contains, 0, 1, _IS_BOOL)
+	ZEND_ARG_TYPE_INFO(0, haystack, IS_STRING, 0)
+	ZEND_ARG_TYPE_INFO(0, needle, IS_STRING, 0)
+ZEND_END_ARG_INFO()
```

Comment:
This is a generated file, please edit `basic_functions.stub.php` instead.

Reply:
Thanks for the hint. I fixed that.

Reply:
It is also important to regenerate basic_functions_arginfo.h by doing `scripts/dev/gen_stub.php basic_functions.stub.php` and to commit the generated file. Otherwise some systems (notably Windows) won't be able to compile (see AppVeyor failures).

Reply:
Sorry for the inconvenience, this is the first time I'm trying to contribute. I've resolved it.

Reply:
No problem. Thanks for contributing! :)